sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -209,7 +212,25 @@ export class AppConfigurationClient {
   listConfigurationSettings(
     options?: ListConfigurationSettingsOptions
   ): Promise<ListConfigurationSettingsResponse> {
-    return this.client.listConfigurationSettings(options);
+
+    if (options == null) {
```

Comment:
Minor: You can make `options` have the default value `{}` where you define your function, then you don't need this check.

Reply:
Oh, that's great. Swapped.